test/node-assertions/test.js

```diff
@@ -0,0 +1,27 @@
+const test = require('@ava/test');
+const exec = require('../helpers/exec');
+
+// The AssertionError constructor in Node 10 depends on the TTY interface
+test('node assertion failures are reported to the console when running in a terminal', async t => {
+	const options = {
+		env: {
+			AVA_SIMULATE_TTY: true,
+			AVA_TTY_COLOR_DEPTH: 8,
+			AVA_TTY_HAS_COLORS: typeof process.stderr.hasColors === 'function'
+		}
+	};
```

Comment:
I don't think this does anything? The TTY simulation is a `test-tap` helper.

Reply:
Right, I am now loading that helper into the ava self hosted tests. As a side note, I had to provide a more complete implementation to the fake TTY stream as [ora](https://github.com/sindresorhus/ora) was [relying](https://github.com/sindresorhus/ora/blob/master/index.js#L232-L236) on them for certain operations.